test/cypress/integration/signup-and-login.spec.js

```diff
@@ -11,6 +11,11 @@ describe('SignUp, Profile and Login', () => {
     cy.giSignUp(userName)
   })
 
+  it('Create group for new user1', () => {
+    cy.giCreateGroup('Dreamers2', 'imageTest.png', 'Testing this user', 420)
```

Comment:
This spec only tests signup and login flows, nothing more.
To test the group creation, use `group-creating-and-invite.spec.js`.
This will make our tests more focused and if one thing fails, it only fails in one place without the need to fix it in multiple places.

Reply:
You can not test any of the rest unless you are part of a group. What do you want me to do? Rename the test suite?

Reply:
Ohhh, True! The easiest solution I see would be to add a link to the profile on the screen as you did to logout and inbox links.